src/abbey/functions/has_attribute.rego

```diff
@@ -0,0 +1,10 @@
+package abbey.functions
+
+import future.keywords.if
+
+# Function which checks whether a user has a given attribute
```

Comment:
nit: Add a period to show this is a complete thought and the new sentence (on the next line) begins.

Reply:
Same nits for other code comments.